source/code-snippets/monitoring/cpm-status.js

```diff
@@ -0,0 +1,32 @@
+function connectionPoolStatus(client) {
+  let checkedOut = 0;
+
+  function onCheckout() {
+    checkedOut++;
+  }
+
+  function onCheckin() {
+    checkedOut--;
+  }
+
+  function onClose() {
+    client.removeListener('connectionCheckedOut', onCheckout);
+    client.removeListener('connectionCheckedIn', onCheckin);
+
+    checkedOut = NaN;
+  }
+
+  // Increases count of active connection pool threads when connectionCheckedIn event is triggered
+  client.on('connectionCheckedIn', onCheckin);
+
+  // Decreases count of active connection pool threads when connectionCheckedOut event is triggered
```

Comment:
I: similar comment as above
```suggestion
  // Increases count of connections checked out of the pool when connectionCheckedOut event is triggered
```